CHANGELOG.md

```diff
@@ -78,6 +78,7 @@
 * [ENHANCEMENT] Include tempo-cli in the release [#1086](https://github.com/grafana/tempo/pull/1086) (@zalegrala)
 * [ENHANCEMENT] Add search on span status [#1093](https://github.com/grafana/tempo/pull/1093) (@mdisibio)
 * [ENHANCEMENT] Slightly improved compression performance [#1094](https://github.com/grafana/tempo/pull/1094) (@bboreham)
+* [ENHANCEMENT] Include metrics for configured limit overrides [#1089](https://github.com/grafana/tempo/pull/1089) (@zalegrala)
```

Comment:
Might be good to mention the metric name here as well.

Reply:
This needs to move up a section as well. it's still in 1.2.